### PR TITLE
Fix force http for lists

### PR DESCRIPTION
**Pull request by Donbur4156**
## About

This pull request is about fixing get() for list when forcing http

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues) (If existent):.
  - resolves #
- I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

File: interactions/utils/get.py [hunk 1]
```diff
@@ -210,30 +210,29 @@ def _check():
...
             _objects.clear()
             _func = getattr(client._http, http_name)
-            for _id in kwargs.get(kwarg_name):
-                _kwargs = kwargs
-                _kwargs.pop(kwarg_name)
-                _kwargs[kwarg_name[:-1]] = _id
+            for _id in kwargs.get(kwarg_names):
+                _kwargs = kwargs.copy()
+                _kwargs.pop(kwarg_names)
+                _kwargs[kwarg_name] = _id
```
Comment (sillyElvy) [inline interactions/utils/get.py]:
```suggestion
            for _id in kwargs.get(kwarg_name):
                _kwargs = kwargs.copy()
                _kwargs.pop(kwarg_name)
                _kwargs[kwarg_name[:-1]] = _id
```

File: interactions/utils/get.py [hunk 2]
```diff
@@ -210,30 +210,29 @@ def _check():
...
             for _index, __obj in enumerate(_objects):
                 if __obj is None:
-                    _id = kwargs.get(kwarg_name)[_index]
-                    _kwargs = kwargs
-                    _kwargs.pop(kwarg_name)
-                    _kwargs[kwarg_name[:-1]] = _id
+                    _id = kwargs.get(kwarg_names)[_index]
+                    _kwargs = kwargs.copy()
+                    _kwargs.pop(kwarg_names)
+                    _kwargs[kwarg_name] = _id
```
Comment (sillyElvy) [inline interactions/utils/get.py]:
```suggestion
                    _id = kwargs.get(kwarg_name)[_index]
                    _kwargs = kwargs.copy()
                    _kwargs.pop(kwarg_name)
                    _kwargs[kwarg_name[:-1]] = _id
```

File: interactions/utils/get.py [hunk 3]
```diff
@@ -186,17 +186,17 @@ def _check():
...
+
+    force_arg = kwargs.pop("force", None)
+    force_cache = force_arg == "cache"
+    force_http = force_arg == "http"
 
     if isinstance(obj, _GenericAlias) or _check():
         _obj: Type[_T] = get_args(obj)[0]
+        http_name = f"get_{_obj.__name__.lower()}"
+        kwarg_name = f"{_obj.__name__.lower()}_id"
+        kwarg_names = kwarg_name + "s"
```
Comment (sillyElvy) [inline interactions/utils/get.py]:
```suggestion
        kwarg_name = f"{_obj.__name__.lower()}_ids"
```

**Review by sillyElvy — CHANGES_REQUESTED** [review]
I don't think there was anything wrong with my scuffed code

**Comment by Donbur4156** [comment]
works this way as well

**Review by sillyElvy — APPROVED** [review]

**Pull request merged**